apps/core/components/userContext/UserContext.js

```diff
@@ -1,18 +1,36 @@
 // @flow
 
 import * as React from 'react';
-import { withContext } from '@kiwicom/margarita-utils';
+import { withContext, noop } from '@kiwicom/margarita-utils';
+import firebase from 'firebase/app';
+import './Firebase';
+
+import { signInWithGoogle } from './SignInHelpers';
 
 type Props = {|
   +children: React.Node,
 |};
 
 type State = {|
+  isFirebaseAvailable: boolean,
   isUserSignedIn: boolean,
+  userEmail: ?string,
+  userDisplayName: ?string,
+  +actions: {|
+    +signIn: () => void,
+    +signOut: () => void,
+  |},
 |};
 
 const defaultState = {
+  isFirebaseAvailable: firebase.apps.length > 0,
```

Comment:
I'm kind of confused by this.

Reply:
To make it clear I've added short description and link to our docs for more info.